specs/DotNetProjectFile.Analyzers.Specs/Rules/MS_Build/Build_actions_should_have_single_task.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Rules.MS_Build.Build_actions_should_have_single_task;
+
+public class Reports
+{
+    [Test]
+    public void empty_nodes()
+        => new BuildActionsShouldHaveSingleTask()
+        .ForProject("MultipleBuildActionTasks.cs")
+        .HasIssues(
+            new Issue("Proj0021", "The Include 'DoesNotExist.cs' of <None> does not exist.").WithSpan(9, 5, 9, 38),
+            new Issue("Proj0021", "The Include '*.vbproj' of <Content> does not contain any files.").WithSpan(10, 5, 10, 43),
+            new Issue("Proj0021", "The Include '**/*.bin' of <EmbeddedResource> does not contain any files.").WithSpan(10, 5, 10, 43),
+            new Issue("Proj0021", "The Include '*.unknown' of <AdditionalFiles> does not contain any files.").WithSpan(12, 5, 12, 43)
```

Comment:
These messages don't seem to match whatever is written in `Rule.cs`

Reply:
Correct. I was working on another rule first (that will be the next after this one) This was a left over, as I was not sure yet about the right way of describing this rule.